### PR TITLE
#3404 extract sensor update actions/reducers

### LINKED ISSUE #3404: Extract bluetooth sensor 'Update' Actions/Reducers
Child of #3380 

## The suggestion

- Currently all bluetooth actions/reducers live in the same place (`VaccineActions`/`VaccineReducers`).
- The suggestion is to extract out specific concerns into their own slices and use `combineReducers` to compose the Bluetooth state.
- This specific issues applies to the following actions:
  - `VACCINE_ACTIONS.SET_LOG_INTERVAL_SUCCESS`
  - `VACCINE_ACTIONS.SET_LOG_INTERVAL_ERROR`
  - `VACCINE_ACTIONS.DISABLE_BUTTON_START`
  - `VACCINE_ACTIONS.DISABLE_BUTTON_STOP`

**Why?**
Clearer separation of concerns, easier state management.

## Describe alternatives you've considered

Leaving as is. But it's getting fairly large now.

## Additional context
N/A

### PR DESCRIPTION
**Pull request by katherine-sussol**
Fixes #3404 

## Change summary

- PR 3 of #3380 to move out update actions. One outstanding to rework the download state.

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Check sensor update commands still work the same (e.g. save new sensor step #3 is one entry point). 

### Related areas to think about
N/A

### CODE REVIEW

File: src/actions/Bluetooth/UpdateActions.js
Comment (joshxg) [inline src/actions/Bluetooth/UpdateActions.js:1]: using this throughout the app as `UpdateActions.set..` doesn't sit right, sorry!

Maybe.. `SensorUpdate` or something..? Am I being ridic?

File: src/reducers/Bluetooth/index.js
Comment (joshxg) [inline src/reducers/Bluetooth/index.js:9]: Yeah.. I dunno what it is. I guess `blink` and `scan` are pretty generic also and they seem fine, and `update` under a `bluetooth` root reducer seems fine- also surely `UpdateActions.setLogInterval` gives enough context ? So I'm not sure what my problem is?

**Review by joshxg — APPROVED** [review]
it's great and I am being silly with naming - gonna approve but see if anyone else thinks the same before merging? 🤷 I dunnooooo

File: src/actions/Bluetooth/UpdateActions.js
Comment (katherine-sussol) [reply src/actions/Bluetooth/UpdateActions.js:1]: Ah, I think I see why it might be a bit weird - there does seem to be a bit of a lack of context when looking at the code without the file structure available? 🤔 . Perhaps we could prefix these all with `Sensor`? Even `Bluetooth/Update` is a little vague thinking about it, could be dealing with Bluetooth headphones or keyboards...or something else.

Comment (joshxg) [reply src/actions/Bluetooth/UpdateActions.js:1]: Sorry didn't think about this before. I suppose generally I don't look at the file structure much, I think that's why I like to export objects like the `XXActions` objects as it gives context to whatever you're importing.

There's certainly a possibility of using bluetooth keyboards and bluetooth barcode scanners in the future, so maybe you're right.

**Comment by katherine-sussol** [comment]
Did a rename to include `Sensor` prefix in the reducer and (hope I caught all the references), does this seem a bit better? 
Open to other suggestions too but I do think it's a bit clearer now.
f4bb111f

File: src/actions/Bluetooth/SensorUpdateActions.js
Comment (joshxg) [inline src/actions/Bluetooth/SensorUpdateActions.js:32]: It IRKS me a bit (I don't know why I thought of irk, but I also had to google it, but what a cool word) that this isn't using the success/error pattern (even though it also IRKS me a bit we aren't doing anything with the error hehe)

File: src/actions/Bluetooth/SensorUpdateActions.js
Comment (joshxg) [inline src/actions/Bluetooth/SensorUpdateActions.js:44]: I just noticed this and it doesn't even matter as we don't seem to be using the payload, but it's a little odd to have `error` sometimes be a string and sometimes be an error object

**Review by joshxg — APPROVED** [review]
Awesome - looks great. I made two silly comments that don't need to be addressed (and shouldn't they're dum!)

**Comment by joshxg** [comment]
Pretty sure I saw you merge something, so believe you have permission, will leave it up to you!

**Comment by joshxg** [comment]
Aaaactually I merged some PRs then there were many conflicts so I fixed them (hopefully ;-) to save time) and I now I don't want that to happen again so I'll just merge now and hopefully I didn't break anything which would make my efforts to save time useless.

🤞 🤞 🤞

**Pull request merged**
